src/utilities/Blacklist.ts

```diff
@@ -0,0 +1,16 @@
+import badWords from '../bad_words/key.json';
+
+interface BadWords {
+  [key: string]: string[];
+}
+
+export const filterBadWords = (input: string, lang: string) => {
+  const badWordsData: string[] = (badWords as BadWords)[lang] ?? badWords.en;
+
+  const regexPattern = badWordsData
```

Comment:
create this regex outside filter function

Reply:
ok. I created